README.md

```diff
@@ -49,35 +49,25 @@ $ cd apisix-dashboard
 
 The `manager-api` is used to provide APIs for Dashboard, just like a bridge between the Apache APISIX and the Dashboard. Here are the steps to build it manually:
 
-1. We need `MySQL/Golang` to be preinstalled.
+1. We need `Go` 1.13+, `ETCD` 3.4+ and `Lua` 5.1+ to be preinstalled.
 
-```sh
-# e.g Initialization for MySQL, please use a more secure Password instead of 123456.
-$ mysql –uroot –p123456
-> source ./api/script/db/schema.sql
-```
-
-2. Start the Apache APISIX.
+2. Check environment variables
 
-[Please follow this guide](https://github.com/apache/apisix#configure-and-installation)
-
-3. Check environment variables
+- enable Go MODULE
+```
```

Comment:
Would better add language here to support code highlight

Reply:
ok